src/include/distributed/multi_join_order.h

```diff
@@ -83,6 +83,24 @@ typedef struct JoinOrderNode
 } JoinOrderNode;
 
 
+/* JoinInfoContext stores joinInfo list and base qualifications */
+typedef struct JoinInfoContext
+{
+	List *baseQualifierList;
+	List *joinInfoList;
+} JoinInfoContext;
+
+
+/* JoinInfoContext stores joinInfo list and base qualifications */
+typedef struct JoinInfo
+{
+	JoinType joinType;
+	uint32 ltableIdx;
+	uint32 rtableIdx;
```

Comment:
would be good to add comments on each field (index into what?) and use our usual long form naming (leftTableIndex)